scripts/govetcheck.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# Check go vet
+echo "==> Checking that the code complies with go vet requirements..."
+govet_out=$(go vet ./... 2>&1)
+if [[ -n ${govet_out} ]]; then
+  echo "go vet has discovered the following issues"
+  echo "${govet_out}"
+  exit 1
```

Comment:
Do we need this script? I think `go vet` exits 1 if it finds anything anyway.

Reply:
You are right we don't need the script. Although the exit code from `go vet` is 2, but I don't think that matters?